### PR TITLE
Clarify rebuilds

### PR DESCRIPTION
**Pull request by paulcsmith**
@jwoertink does this make more sense?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,10 +4,14 @@ Up makes it easier to start, build, and run Docker for local development.
 
 * **Much shorter commands.** For example, `up node test` instead of
   `docker-compose up run --rm web node test`
-* **Automatic image rebuilds.** No more pulling from git and wondering why your
-  project isn't working. Up will track necessary files and rebuild for you.
+* **Automatic image rebuilds.** Up will rebuild containers when you run
+  a command and files your images depend on are changed. No more pulling
```
Comment (jwoertink) [inline README.md]: This reads a little weird.

> Up will rebuild containers when you run a command and change files that your images depend on.

Maybe something like that?

File: README.md [hunk 2]
```diff
@@ -4,10 +4,15 @@ Up makes it easier to start, build, and run Docker for local development.
 
 * **Much shorter commands.** For example, `up node test` instead of
   `docker-compose up run --rm web node test`
-* **Automatic image rebuilds.** No more pulling from git and wondering why your
-  project isn't working. Up will track necessary files and rebuild for you.
+* **Automatic image rebuilds.** When files your image depends on are changed 
+  (`package.json`, `Dockerfile`, etc.), Up will  rebuild the images before
+  running commands. No more pulling from git and wondering why your project 
+  isn't working (oops, forgot to run `docker-compose build`).
```
Comment (paulcsmith) [inline README.md]: How is this? Added some examples to maybe clear it up a bit more

Comment (jwoertink) [reply README.md]: nice! Yeah, that makes a lot more sense.

Comment (paulcsmith) [reply README.md]: Awesome! Thanks for the feedback!

**Pull request merged**